templates/etc/icinga2/scripts/twilio-sms-service-notification.sh.j2

```diff
@@ -30,7 +30,7 @@ Required parameters:
 
 Optional parameters:
   -i ICINGAWEB2URL (\$notification_icingaweb2url\$, Default: unset)
-  -r USERPHONE (\$user.pager\$)
+  -S SLA_TEAM (\$notification_slateam\$, Default: unset)
```

Comment:
The `SLA_` prefix might be pointless. Thinking about removing it.

In fact, we can maybe even generalise this option to just take "any further information", to be used in the user-provided `icinga2_master_twilio_sms_additional_commands`.

Reply:
`-X` for "extra information" comes to mind. Any better ideas/suggestions? :slightly_smiling_face:

Reply:
Now reading `-X` instead of `-S`, and renamed the corresponding variable to `EXTRA_DATA`.